lib/jekyll-import/importers/medium.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module JekyllImport
+  module Importers
+    class Medium < Importer
+      def self.specify_options(c)
+        c.option "username",       "--username NAME",  "Medium username"
+        c.option "canonical_link", "--canonical_link", "Copy original link as canonical_url to post (default: false)"
+        c.option "render_audio",   "--render_audio",   "Render <audio> element in posts for the enclosure URLs (default: false)"
+      end
+
+      def self.validate(options)
+        abort "Missing mandatory option --username." if options["username"].nil?
+      end
+
+      def self.require_deps
+        Importers::RSS.require_deps
+      end
+
+      def self.process(options)
+        Importers::RSS.process({
+          "source"         => "https://medium.com/feed/@#{options.fetch("username")}",
+          "render_audio"   => options.fetch("render_audio", false),
```

Comment:
Do Medium articles typically have audio?

Reply:
Till now I haven't found any post with audio option. But if I am not wrong, it's possible to add video in medium post